apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -925,3 +927,26 @@ with_log(Fun, Msg) ->
         {error, Reason} ->
             emqx_ctl:print("~s FAILED~n~p~n", [Msg, Reason])
     end.
+
+cluster_info() ->
+    RunningNodes = safe_call_mria(running_nodes, [], []),
+    StoppedNodes = safe_call_mria(cluster_nodes, [stopped], []),
+    #{
+        running_nodes => RunningNodes,
+        stopped_nodes => StoppedNodes
+    }.
+
+%% CLI starts before mria, so we should handle errors gracefully:
+safe_call_mria(Fun, Args, OnFail) ->
```

Comment:
I am afraid this function will be abused, Ignoring the exception is well for `get status`, but may be dangerous for other operations

Reply:
It's just a UI module, it cannot perform any meaningful error handling or recovery anyway. Could you give an example situation where catching errors in the CLI module would be dangerous?